src/servicehost/servicehost/Program.cs

```diff
@@ -11,9 +13,37 @@ public static void Main(string[] args)
             {
                 servicehost.Start(cli.Endpoint);
 
-                Console.WriteLine("Service host running...");
-                Console.ReadLine();
+                Console.WriteLine("Service host running on " + cli.Endpoint.ToString());
+
+                if (IsRunningOnMono())
+                {
+                    var terminationSignals = GetUnixTerminationSignals();
+                    UnixSignal.WaitAny(terminationSignals);
```

Comment:
Führt das dazu, dass unter mono auch in der konsole gewartet wird wie mit ReadLine()?

Warum die signale? Damit in dropstack kontrolliert heruntergefahren werden kann, auch wenn niemand an einer tastatur sitzt?